docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -31,6 +31,7 @@ We conduct testing and ensure compatibility against the following OpenShift vers
 
 | OpenShift Version | [End of Support Date](https://access.redhat.com/support/policy/updates/openshift) |
 | ----------------- | --------------------------------------------------------------------------------- |
+| 4.18.x            | GA of 4.19 + 3 Months                                                             |
 | 4.17.x            | June 27, 2025                                                                     |
 | 4.16.x            | December 27, 2025                                                                 |
 | 4.15.x            | August 27, 2025                                                                   |
```

Comment:
maybe I'm reading the reference wrong but

4.15 / 4.16 references the `maintenance support ends`

4.17 references god knows what 🤷, it's `April 1, 2026` if we take maintenance support.
4.18 already has `August 25, 2026` defined.

Reply:
perfect, either way works 👍